package.json

```diff
@@ -1,14 +1,14 @@
 {
   "name": "dbstream-memory",
-  "version": "1.1.3",
+  "version": "1.2.0",
```

Comment:
Looks like the `package-lock.json` file had been generated before you changed the version here:

<img width="499" alt="Screen Shot 2022-08-31 at 6 50 03 PM" src="https://user-images.githubusercontent.com/43310906/187722598-89a42044-dc1f-4a38-8d31-3952a0d03a9f.png">